docs/blog/2021-07-04-ux-case-studies-intro.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-slug: ux-study-timeline
+slug: ux-studies-intro
```

Comment:
Sorry that I can't remember, if you were planning on adding to this to make it a long form, or if the case-studies will be seperate posts?

I'm asking here because maybe a slug of just `ux-studies` might give you flexibility later on as it doesn't imply a series, but you could still do `ux-studies-part2` etc later on. I'm only saying this because changing the title and the content later is fine, but we don't want to change the slug after publish.

But than again, if you've decided hard on a series, that cool and no need to change it.